Azure/sonic-buildimage
# [docker-mgmt-framework]: update mgmt framework docker to support sonic-cli cmd

**- Why I did it**
- migrate to python3 support
- add dependent packages for Klish 
- allow login as non-root user

**- How I did it**
update sonic-cli script to start Klish with user name, system name and timeout
update the Dockerfile.j2 to resolve dependent packages
add python3-dev for Klish use 

**- How to verify it**
Incremental buster build with https://github.com/Azure/sonic-mgmt-framework/pull/76 and verify the sonic-cli

**- Which release branch to backport (provide reason below if selected)**

**- Description for the changelog**
Migrate to python3.7 support, update sonic-cli script and resolve package dependencies 

**- A picture of a cute animal (not mandatory but encouraged)**
